demo/yun/pages/posts/date.md

```diff
@@ -2,4 +2,73 @@
 title: Date test
 date: 2023-07-19 18:55:53
 # updated: 2023-07-19 18:55:53
+toc: false
 ---
+
+<FormatDate format="MMM d" />
```

Comment:
I think we need a test for these.

Reply:
Here, I emphasize more on the usage of the tool rather than prioritizing testing. I'm not sure how to perform comprehensive testing in this context. The time zone is determined by the user's time zone, so if the answer is fixed, the test results will vary in different regions. I have validated the correctness of the output for a long time and have thought a lot about this matter. Since this question has been raised, I need to consider how to proceed in the future.

Reply:
Here we can add an end-to-end test to make sure the `formatDate` is displayed correctly and in the form we expect.

Just like this: https://github.com/YunYouJun/valaxy/blob/b832ccadb3aa9d7a96c9fc9a9eabaf2d7e9dca1a/e2e/theme-yun/fm.spec.ts#L16

Reply:
ok, I will try it later today